auth0-authentication/README.md

```diff
@@ -84,7 +81,19 @@ as of now, the `graphcool-lib` npm module is not available in webtask.io yet.
   ```sh
   wt create auth0-authentication.js --secrets-file auth0-authentication.js.secrets
   ```
-* Copy webtask URL from console and continue with setting up a Schema Extension in Graphcool Console
+* Because the `package.json` in this folder specifies `graphcool-lib` as dependency, and this dependency is not available by default on Webtask.io, you will see the following output:
+  ```sh
+  * Hint: A package.json file has been detected adjacent to your webtask. Ensuring that all dependencies from that file are avialable on the platform. This may take a few minutes for new versions of modules so please be patient.
+  * Hint: If you would like to opt-out from this behaviour, pass in the --ignore-package-json flag.
+  Resolving 1 module...
+  Provisioning 1 module...
+  graphcool-lib@0.0.3 is available
+  Webtask created
+
+  You can access your webtask at the following url:
+
+  https://__YOUR_WEBTASK_URL__```
+* Copy the webtask URL from console and continue with setting up a Schema Extension in Graphcool Console
```

Comment:
In case of Webtask it is necessary to append `/authenticate` to webtask URL in order for the schema extension function to work correctly, because embedded express server listens on this path inside the function.

With this modification in instructions, the webtask function works correctly 👍 .

Thanks for your help 🙂 .